apps/enc.c

```diff
@@ -51,6 +51,7 @@ typedef enum OPTION_choice {
 const OPTIONS enc_options[] = {
     {"help", OPT_HELP, '-', "Display this summary"},
     {"ciphers", OPT_LIST, '-', "List ciphers"},
```

Comment:
It seems it's OK to change the description of `-ciphers` from "List ciphers" to something like "Alias of -list" to make it more canonical.